packages/medusa/src/strategies/cart-completion.ts

```diff
@@ -409,6 +365,76 @@ class CartCompletionStrategy extends AbstractCartCompletionStrategy {
       response_body: { data: order, type: "order" },
     }
   }
+
+  private async confirmAndUpdateInventory(
```

Comment:
self-review: @olivermrbl I did those changes as a scout action while debugging the integration tests. It helped me have a clearer view of the surroundings instead of having this big block inside the payment-authorized method. Are you okay with keeping it or prefer a revert of that file?

Reply:
I prefer we revert tbh. Don't want this to be bigger than it already is 😅

Reply:
Got it lake sense 🙌